conf/machine/include/imx8mn-evk.inc

```diff
@@ -38,14 +42,15 @@ SERIAL_CONSOLES = "115200;ttymxc1"
 BOOT_SPACE = "65536"
 LOADADDR = ""
 UBOOT_SUFFIX = "bin"
-UBOOT_MAKE_TARGET = ""
```

Comment:
Couldn't the variable here be dropped and use the default used in `imx-base.inc`?

Reply:
_imx-base.inc_ sets it as default to:
```python
UBOOT_MAKE_TARGET ?= "u-boot.${UBOOT_SUFFIX}"
```

There are 2 overrides next to this assignment: one for `mxs` and one for `mx8`, which would not apply here as they are filtered out.

Besides, there are several derivatives, which are setting the `UBOOT_MAKE_TARGET` to empty string, and I'm not sure that change to "all" would not break them.

And (as a cherry on top of the pie): `imx8qm-mek` and `imx8x-mek` have it assigned conditionally based on whether `optee` feature is enabled or not.

I'd prefer to keep the definition of the U-Boot build target as it is now, changing it to all as default in _imx-base.inc_ might break things.

-- andrey

Reply:
That's fine. I want to clean it later but we ought to have something merged first.